Core/MIPS/MIPSIntVFPU.cpp

```diff
@@ -629,18 +629,24 @@ namespace MIPSInt
 			// vsat0 changes -0.0 to +0.0, both retain NAN.
 			case 4: if (s[i] <= 0) d[i] = 0; else {if(s[i] > 1.0f) d[i] = 1.0f; else d[i] = s[i];} break;    // vsat0
 			case 5: if (s[i] < -1.0f) d[i] = -1.0f; else {if(s[i] > 1.0f) d[i] = 1.0f; else d[i] = s[i];} break;  // vsat1
-			case 16: d[i] = 1.0f / s[i]; break; //vrcp
+			//case 16: d[i] = 1.0f / s[i]; break; //vrcp
```

Comment:
Let's just remove the old cases.  At most it could be:

```
			case 16: { d[i] = vfpu_rcp(s[i]); } break; //vrcp (1.0f / s[i])
```

So it's clear what it does, but I don't think we need to keep the old code around.

-[Unknown]